doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1894,22 +1906,34 @@ Response:
                 {
                     "type": "Room",
                     "id": "Bcn-Welt",
-                    "temperature": 21.7,
-                    "humidity": 60
+                    "temperature": {
+                      "value": 21.7,
+                      "type": "none"
+                     },
+                    "humidity": {
+                      "value": 60,
+                      "type": none
+                    }
                 },
                 {
                     "type": "Room",
                     "id": "Mad_Aud",
-                    "temperature": 22.9,
-                    "humidity": 85
+                    "temperature": {
+                      "value": 22.9,
+                      "type": "none"
+                    },
+                    "humidity": {
+                      "value": 85,
+                      "type": "none"
+                    }
                 }
             ]
         }
 
 + Parameters
     + options (optional, string) - Options dictionary
       + Members
-          + keyValues - when used, the request payload uses `keyValues` simplified entity representation. See corresponding 
+          + keyValues - when used, the request payload uses `keyValues` simplified entity representation. See "Simplified Entity Representation"
```

Comment:
I'm not paying attention to that in this PR (doing so could add some "diff-noise", making review harder), but I think it is very timely comment for the proof-read PR. @kzangeli please take note of this.